### PR TITLE
header and footer changes

### PR DESCRIPTION
**Pull request by dahfool**
Header and footer changes, including profile/ register and new links

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -249,13 +270,13 @@ <h3 class="h3">Grids & Breakpoints</h3>
         <div class="outer-container">
           <div class="footer-section">
             <div class="col-8">
-              <a href="https://www.exportingisgreat.gov.uk/" title="Go to the Exporting is Great homepage">
-                <img src="/images/EiG-logo-stacked.svg" alt="Exporting is Great logo" class="footer-logo">
+              <a href="https://www.exportingisgreat.gov.uk/" title="Go to the Exporting is GREAT homepage">
+                <img src="/images/EiG-logo-stacked.svg" alt="Exporting is GREAT logo" class="footer-logo">
```
Comment (appility) [inline public/index.html]: it's a small thing but I recently had some issues with case-insensitivity in files on AWS buckets - could we lower case all filenames?

File: public/index.html [hunk 2]
```diff
@@ -249,13 +270,13 @@ <h3 class="h3">Grids & Breakpoints</h3>
         <div class="outer-container">
           <div class="footer-section">
             <div class="col-8">
-              <a href="https://www.exportingisgreat.gov.uk/" title="Go to the Exporting is Great homepage">
```
Comment (appility) [inline public/index.html]: This is the old url - the new one will be export.great.gov.uk

File: public/images/icons/signout.svg [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
```
Comment (appility) [inline public/images/icons/signout.svg]: We could probably lint some of the cruft out of these svgs to optimize them

File: public/index.html (hunk 2 above)
Comment (dahfool) [inline public/index.html]: @appility Seems like export.great.gov.uk currently redirects to https://www.exportingisgreat.gov.uk/. I will ask more clarification though.

Comment (appility) [inline public/index.html]: yes, the DNS changes are scheduled for today and on the 31st https://www.exportingisgreat.gov.uk/ will be using redirects to export.great.gov.uk

File: public/index.html (hunk 1 above)
Comment (dahfool) [inline public/index.html]: @appility I will rename the files.

File: public/images/icons/signout.svg (hunk 3 above)
Comment (dahfool) [inline public/images/icons/signout.svg]: @appility Other teams have image optimization as part of their built or deploy progress. This is something we might implement for the pattern library in the future.

**Comment by dahfool** [comment]
@appility all images have been rename to be lowercased and changed https://www.exportingisgreat.gov.uk/ to www.export.great.gov.uk

**Comment by appility** [comment]
Great

**Pull request merged**